working/templates/SharedItems/SkylineDevopsHelp/GettingStarted.md

```diff
@@ -0,0 +1,119 @@
+# Getting Started with Skyline DataMiner DevOps
+
+Welcome to the Skyline DataMiner DevOps environment!  
+This quick-start guide will help you get up and running. It was auto-generated based on the initial project setup during creation.  
+For more details and comprehensive instructions, please visit [DataMiner Docs](https://docs.dataminer.services/).
+
+<!--#if (CreateDataMinerPackage)-->
+## Creating a DataMiner Application Package
+
+This project was configured to create a `.dmapp` file every time you build the project.  
+Simply compile or build the project, and you will find the generated `.dmapp` in the standard output folder, typically the `bin` folder of your project.
+
+When Publishing, this project will become its own item on the online catalog.
+
+<!--#else-->
+## Enabling the Creation of a DataMiner Application Package
+
+This project was not configured to generate a `.dmapp` file.  
+To enable `.dmapp` generation, consider migrating to a multi-artifact DataMiner Application Package as described below.
+
+<!--#endif-->
+## Migrating to a Multi-Artifact DataMiner Application Package
+
+If you need to combine additional components in your `.dmapp` file, you should:
+
+1. Open the `$SCRIPTNAME$.csproj` file and ensure the `<GenerateDataminerPackage>` property is set to `False`.
+2. Right-click your solution and select **Add > New Project**.
+3. Select the **Skyline DataMiner Package Project** template.
+
+Follow the provided **Getting Started** guide in the new project for further instructions.
+
+<!--#if (IsCatalogNoCICD)-->
+## Publishing to the Catalog
+
+This project was created with support for publishing to the DataMiner catalog.  
+You can publish your artifact either manually via the Visual Studio IDE or by setting up a CI/CD workflow.
+
+### Manual Publishing
+
+1. Obtain an **Organization Key** from [admin.dataminer.services](https://admin.dataminer.services/) with the following scopes:
+   - **Register catalog items**
+   - **Read catalog items**
+   
+2. Securely store the key using Visual Studio User Secrets:
+   - Right-click the project and select **Manage User Secrets**.
+   - Add the key in the following format:
+
+```json
+{
+  "skyline": {
+    "sdk": {
+      "catalogpublishtoken": "MyKeyHere"
+    }
+  }
+}
+```
+
+3. Publish the package using the **Publish** option in Visual Studio.
+
+**Recommendation:** For stable releases, consider using a CI/CD setup to run **dotnet publish** only after passing quality checks.
+
+<!--#elseif (IsCatalogBasicCICD)-->
+## Publishing to the Catalog with Basic CI/CD Workflow
+
+This project was created with a basic GitHub workflow for catalog publishing.  
+Follow these steps to get started:
+
+1. Create a GitHub repository and push your project.
+2. The first workflow run will fail due to missing secrets. View the error details on the **Actions** page of your repository.
```

Comment:
```suggestion
2. If the first workflow run fails because of missing secrets, check the error details on the **Actions** page of your repository and resolve the issues.
```

Reply:
This was rewritten with more details based on user feedback.